### PR TITLE
Small code refactoring

### PR DESCRIPTION
**Pull request by ya-ilya**
- Fixed some paddings
- Fixed some modifiers for fields and constructors
- Fixed some comments (`<p>`)
- Some `if` replaced by ternary operator
- Removed unused imports and variables
- Removed raw class usage

### CODE REVIEW

File: src/main/java/me/earth/earthhack/impl/modules/misc/tracker/ListenerTick.java [hunk 1]
```diff
@@ -20,99 +20,96 @@ public ListenerTick(Tracker module)
     @Override
     public void invoke(TickEvent event)
     {
-        if (event.isSafe())
+        if (event.isSafe() && module.isEnabled())
```
Comment (cattyngmd) [inline src/main/java/me/earth/earthhack/impl/modules/misc/tracker/ListenerTick.java:23]: Can you remove `module.isEnabled()` check? The method wont be invoked if the module is disabled.

**Review by cattyngmd — APPROVED** [review]

**Review by cattyngmd — APPROVED** [review]
👍

**Review by 3arthqu4ke — APPROVED** [review]
LGTM!

**Comment by 3arthqu4ke** [comment]
I will merge once this https://github.com/3arthqu4ke/3arthh4ck/actions/runs/2940205955 is green

**Pull request merged**